fishy/engine/semifisher/fishing_event.py

```diff
@@ -32,9 +34,16 @@ class FishEvent:
     uid = None
     sound = False
 
+def _fishing_sleep(waittime, lower_limit = 16, upper_limit = 58):
```

Comment:
16-58 ms seem too less, try to find out the max time we can delay reeling in the fish, then add an offset just so the fish doesn't escapes sometimes, use those values (I'm not sure if you are already doing that 😅 ignore this if you are)

Reply:
I can try that, will take a moment though. But I will only trial and error this by hand, so the values won't be to sharp. From my testings I think 16-58ms are very human like. Thats about the time it takes me to click, as long as I am attentive.

Reply:
yeah but imagine, people wont be that attentive when they are fishing for hours 😉